src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+    padding: 0;
+  }
+
+  &__productName {
+    color: $main-accent;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    width: 166px;
+    margin: 0 16px 4px;
```

Comment:
Make inline margins in some container instead of repeat it in each element of card